chainer/backends/cuda.py

```diff
@@ -541,12 +546,9 @@ def get_array_module(*args):
         the arguments.
 
     """
-    if available:
-        args = [arg.data if isinstance(arg, chainer.variable.Variable) else arg
-                for arg in args]
-        return cupy.get_array_module(*args)
-    else:
-        return numpy
+    warnings.warn('backends.cuda.get_array_module is deprecated. Please use '
```

Comment:
I think it's too early to emit warning, considering that this API is used in many places. How about doing it in v6?

Reply:
Thanks for the suggestion. I also think that's okay so removed the warning.